beacon_node/rpc/src/attestation.rs

```diff
@@ -136,11 +138,10 @@ impl<T: BeaconChainTypes> AttestationService for AttestationServiceInstance<T> {
                     "type" => "valid_attestation",
                 );
 
-                // TODO: Obtain topics from the network service properly.
-                let topic = types::TopicBuilder::new("beacon_chain".to_string()).build();
+                // valid attestation, propagate to the network
+                let topic = TopicBuilder::new(SHARD_TOPIC_PREFIX).build();
```

Comment:
It might be a simpler and more separated API if we just had `Topic::shard()` instead of `TopicBuilder::new(SHARD_TOPIC_PREFIX).build();`?

Just a thought for the future.

Reply:
Yeah, TopicBuilder comes from libp2p. I agree.. Will make a note and build a wrapper for shards in future revisions